devicetypes/rooms-beautiful/rooms-beautiful-curtain.src/rooms-beautiful-curtain.groovy

```diff
@@ -167,93 +218,76 @@ private handleBatteryEvent(volts) {
 
 			sendEvent(name: "battery", value: percent)
 			sendEvent(name: "battLife", value: value)
-			log.debug "${linkText} - Batt: ${value} **** Volts: ${volts/10}v **** Percent: ${percent}%"
+			log.info "${linkText} - Batt: ${value} **** Volts: ${volts/10}v **** Percent: ${percent}%"
 		}
 	}
 }
 
 def refresh() {
 	zigbee.onOffRefresh() +
-	zigbee.readAttribute(0x0001, 0x0020) +
+    zigbee.readAttribute(0x0102, 0x0008) 	// Window Lift Percentage Attribute
+	zigbee.readAttribute(0x0001, 0x0020) +	// Battery Voltage Attribute
```

Comment:
Better to use `zigbee.POWER_CONFIGURATION_CLUSTER` than the hardcoded cluster ID